db/migrate/20151012175123_add_times_purchased_to_books.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimesPurchasedToBooks < ActiveRecord::Migration
+  def change
+     add_column :books, :times_purchased, :integer
+  end
+end
```

Comment:
I would remove this as per [my comment in the view above](https://github.com/jesseract/bookstore/pull/3/files?diff=split#diff-b9cabdc15aa47777492ecb56db9bfac3R25)